docker/test/test_app.py

```diff
@@ -0,0 +1,99 @@
+from tqdm import tqdm
+import os
+import json
+import requests
+
+from constants import ACTION_INVOKED_GROUP, Method, METHODS_CNT
+from reader import event_to_tips
+
+
+TEST_EVENTS_DIR = "./test_events"
+TEST_LABELS_DIR = "./test_labels"
+FILE_NAMES = os.listdir(TEST_EVENTS_DIR)
+
+
+def _read_user_events(file):
+    with open("./test_events/" + file, 'r') as fin:
```

Comment:
Shouldn't we put under VCS the files required for the tests?

Reply:
The files can be generated using "generate_test_files.py", but I added some, as an example